### ISSUE TITLE
Catch/improve "The node to be removed cannot be identical to the node on which the removal is initiated"

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#1202]
We performed a `remove node` on the node that will be removed.

We got this error message:
```
root@ovs-node01-1604:~# ovs remove node 10.100.199.181
+++++++++++++++++++++
+++  Remove node  +++
+++++++++++++++++++++
WARNING: Some of these steps may take a very long time, please check the logs for more information


Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "ovs/lib/setup.py", line 627, in remove_node
    raise RuntimeError('The node to be removed cannot be identical to the node on which the removal is initiated')
RuntimeError: The node to be removed cannot be identical to the node on which the removal is initiated
```

**Comment by wimpers** [comment]
@kinvaris  what would you expect as behaviour?

**Comment by khenderick** [comment]
I highly suspect that @kinvaris expects a slightly nicer message to appear; meaning catching the error and printing it without stack trace. I agree with that.

**Comment by khenderick** [comment]
Fixed by #1213, packaged in openvstorage-2.7.6-rev.4357.22393a7

**Comment by JeffreyDevloo** [comment]
### Steps
- Setup cluster
- Configured cluster
- Removed the node where the removal was executed

### Output
```
root@ovs-node-3:~# ovs remove node 10.100.199.153
+++++++++++++++++++++
+++  Remove node  +++
+++++++++++++++++++++
WARNING: Some of these steps may take a very long time, please check the logs for more information


++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+++  The node to be removed cannot be identical to the node on which the removal is  +++
+++  initiated                                                                       +++
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

```
### Test result
No error message was displayed.
Test passed.

### Packages
- openvstorage                         2.7.7-rev.4361.7c05190-1        amd64        openvStorage
- openvstorage-backend                 1.7.7-rev.828.5e3746c-1         amd64        openvStorage Backend plugin
- openvstorage-backend-core            1.7.7-rev.828.5e3746c-1         amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps         1.7.7-rev.828.5e3746c-1         amd64        openvStorage Backend plugin Web Applications
- openvstorage-core                    2.7.7-rev.4361.7c05190-1        amd64        openvStorage core
- openvstorage-hc                      1.7.7-rev.828.5e3746c-1         amd64        openvStorage Backend plugin HyperConverged
- openvstorage-sdm                     1.6.7-rev.455.7913d38-1         amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps                 2.7.7-rev.4361.7c05190-1        amd64        openvStorage Web Applications
